README.md

```diff
@@ -0,0 +1,108 @@
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3.org/html/">HTML</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/Overview.en.html">SCSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+- **Linters**: [Lighthouse](https://developer.chrome.com/docs/lighthouse/overview/), [Webhint](https://webhint.io/), and [Stylelint](https://stylelint.io/.)
+- **A `.github/workflows/linters.yml`** file to run linters on GitHub Pull Request.
+- **A** `.gitignore` file including `node_modules/`.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> A new developer or learning student could use this as a sample to complete a similar project.
+
+To get a local copy up and running, follow these steps.
+### Prerequisites
+
+In order to run this project you need:
+
+- Have [npm](https://www.npmjs.com/package/npm) installed in your project:
+  `curl -qL https://www.npmjs.com/install.sh | sh`
+
+### Setup
+
+Clone this repository to your desired folder:
+
+`git clone git remote add origin https://github.com/ClintonYounge/My-Portfolio-Project`
+
+or
+
+`git clone git remote add origin https://github.com/ClintonYounge/Hello-Microverse-Project` <-- _Only if you have [SSH keys configured](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) on your GitHub account_.
+
+or
+
+Utilizing Github Desktop. A simple interface, thats powerful enough to simplify this process.
+https://desktop.github.com/
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Clinton Younge**
+
+- GitHub: [@ClintonYounge](https://github.com/ClintonYounge)
+- Twitter: [@YoungeCjay](https://twitter.com/YoungeCjay)
+- LinkedIn: [@clintonyounge](https://www.linkedin.com/in/clinton-younge-83386a25a/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my standup team for help settling my mind to persevere through this project.
+
+I also want to thank to my Brother *Charles Gardimy* for his help, advice and encouragement to press onward through the tough times.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] This link is broken and is not working. kindly consider changing `(./LICENSE)` to `(./MIT.md)`. it will fix your link.